src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package calculator;
+
+import calculator.element.Element;
+import calculator.element.operand.Operand;
+import calculator.element.operaor.*;
+import calculator.parser.InputParser;
+import calculator.parser.ElementDeque;
+
+public class Calculator {
+
+    static Integer equality(String input) {
+        ElementDeque elementDeque = InputParser.parseResult(input);
+        return operateMultiple(elementDeque);
+    }
+
+    private static Integer operateMultiple(ElementDeque elementDeque) {
+        while(! elementDeque.isQuitCalculate()) {
+            Operand operandA = (Operand) elementDeque.popElement();
```

Comment:
```suggestion
            Operand operandA = elementDeque.popOperand();
            Operator operator = elementDeque.popOperator();
            Operand operandB = elementDeque.popOperand();
```
캐스팅을 매번 클라이언트에서 처리하기 보단 dequeue에서 처리해주면 어떨까요

Reply:
예리하시네요. 어글리한 부분인거 같습니다. 바로 변경할게요.